controllers/operatorconfigmap_controller.go

```diff
@@ -654,31 +670,31 @@ func (c *OperatorConfigMapReconciler) reconcileSubscriptionValidatingWebhook() e
 }
 
 func (c *OperatorConfigMapReconciler) reconcileClientOperatorSubscriptionLabel() error {
```

Comment:
We are not reconciling a label, we are reconciling a subscription 

```suggestion
func (c *OperatorConfigMapReconciler) reconcileClientOperatorSubscription() error {
```